### DISCUSSION TITLE
Check for null : Test Filters

### DISCUSSION THREAD

**Discussion by nikhilp79** [thunderclient/thunder-client-support#1122] [Q&A] [answered]
I am trying to get the count of objects where "ss" is null from this response. I tried `json.results[ss = null] | length` but that doesn't give me correct results. Also, is it possible to get the count of objects with property 'ss' (either `null `or some value ; last element doesn't have 'ss' property) using built-in filters ? Thanks in advance!
```
{
	"results": [
		{
			"id": 3,
			"name": "a",
                        "ss" :"123-32-A2"
		},
		{
			"id": 4,
			"name": "d",
                        "ss" : null
		},
		{
			"id": 3,
			"name": "Z",
                        "ss" :"125-32-DD"
		},
		{
			"id": 8,
			"name": "no name"
		}
	]
}
```

**Comment by rangav** [comment, marked as answer]
Hi @nikhilp79 current check for null is not working

but you can use custom filter to test this use case and write you own js logic

In Json query you can use: `json.results | customFilter1`

**Reply by rangav** [reply]
Custom Filter Docs
https://github.com/rangav/thunder-client-support/blob/master/docs/filters.md#custom-filters
